### PR TITLE
wiki(Security): Add a security policy

### PR DESCRIPTION
**Pull request by Kitzunu**
Maybe this is a bit of a reach, or it is good to have?

You tell me :D

### CODE REVIEW

**Comment by Winfidonarleyan** [comment]
About support `MariaDB`
https://github.com/azerothcore/azerothcore-wotlk/blob/d1e83f6e3d6255802e123845f3377685d9b78695/src/cmake/macros/FindMySQL.cmake#L31

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+| ------------------ | ------------------ |
+| master             | :white_check_mark: |
+
+We support these versions of MySQL:
+
+| MySQL Version | Supported          |
+| ------------- | ------------------ |
+| 5.6           | :white_check_mark: |
+| 5.7           | :white_check_mark: |
+| 8.0           | :x:                |
```
Comment (FrancescoBorzi) [inline SECURITY.md:17]: soon will be supported

Comment (Kitzunu) [reply SECURITY.md:17]: Yep, but so far not :D

**Comment by Kitzunu** [comment]
@Yehonal Thoughts? <3

**Comment by Yehonal** [comment]
Thanks for this contribution :)

**Comment by Yehonal** [comment]
Shouldn't be linked somewhere?

**Comment by Yehonal** [comment]
However, basically we still support the version 2.x (the branch) even though no one is using it practically I guess, but if someone discover a security issue there or something like that, we should fix that security issue on both master and 2.x (cherry pick)

Moreover, is it related to the requirement page?

**Comment by Kitzunu** [comment]
Just https://github.com/azerothcore/azerothcore-wotlk/security

**Comment by BarbzYHOOL** [comment]
how is it related to security though?

it's totally unrelated (imo), the mariadb versions etc...

**Pull request merged**